src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequence.tsx

```diff
@@ -64,6 +70,9 @@ const BlastInputSequence = (props: Props) => {
   const deleteButtonRef = useRef<HTMLButtonElement | null>(null);
   const textareaRef = useRef<HTMLTextAreaElement | null>(null);
 
+  const errorTooltipDescription =
+    'Please check that all your sequences are nucleotide or protein, and that they do not contain any invalid characters';
```

Comment:
let's move this out of the component function

Reply:
I have moved it out but wasnt sure where else to put it so i have made it constant just outside the component function. let me know if there is a better place for it.

Reply:
This should now be moved...